jets/e/secp.c

```diff
@@ -0,0 +1,356 @@
+/* j/5/secp.c
+**
+*/
+#include "all.h"
+#include "../include/secp256k1.h"
+#include "../include/secp256k1_recovery.h"
+
+// end-to-end test:
+//   =hash 123
+//   =priv 99
+//   =nonce (make-k:secp256k1:secp:crypto `@uvI`hash priv)
+//   =sig (ecdsa-raw-sign:secp256k1:secp:crypto `@uvI`hash priv)
+//   =reco (ecdsa-raw-recover:secp256k1:secp:crypto `@uvI`hash v:sig  r:sig s:sig)
+
+
+
+
+// util funcs
+//
+
+#if 0
+static void pretty_print_hex(char * headline,
```

Comment:
Now that this jet has been implemented and tested, will this ever be used again? If not, probably best to just take it out. You can remove it in a separate commit if you care about it being retrievable from git history.

Reply:
O_O

>  will this ever be used again?

I don't know.  Is there a chance that we will ever work with binary blobs in the future?  Or ever find a bug in our code?

Anyway: done.